doc/RESTAPI_Guidelines.md

```diff
@@ -122,3 +122,52 @@ the key, its better to either create a new key for that purpose or expose a new
 version of the api and modify the key contents in the new version.
 </pre>
 </details>
+
+# JSON Error Response Schema :
+
+To be consistent in returning errors(when using REST API) with other components
+such as S3, we will follow the error format that S3 currently uses 
+(Schema shown below).
+
+<details>
+<summary>Schema</summary>
+<pre>
+
+>{
+>  "$schema": "http://json-schema.org/draft-04/schema#",
```

Comment:
One suggestion : the indentation of this json object is not seen when we see the actual document. Which makes it difficult to read.  Can we change the format from `>` to ` ``` `  ?

Reply:
Done